net/gce.sh

```diff
@@ -119,6 +120,10 @@ while getopts "h?p:Pn:c:z:ga:d:" opt; do
     enableGpu=true
     leaderMachineType="$gpuLeaderMachineType"
     ;;
+  G)
+    enableGpu=true
+    leaderMachineType="$cpuLeaderMachineType --accelerator $OPTARG"
```

Comment:
Let's make this `leaderMachineType="$OPTARG"`

This means you need to pass a little more on the CLI but avoids encode GCE-specifies (this part of script is also used for AWS).

Reply:
Done